pandas/tests/arrays/sparse/test_arithmetics.py

```diff
@@ -44,9 +46,27 @@ def _check_numeric_ops(self, a, b, a_dense, b_dense, mix, op):
 
             if op in [operator.floordiv, ops.rfloordiv]:
                 # Series sets 1//0 to np.inf, which SparseArray does not do (yet)
-                mask = np.isinf(expected)
-                if mask.any():
-                    expected[mask] = np.nan
+                condition = _np_version_under1p20
+                if isinstance(a_dense, np.ndarray) and isinstance(b_dense, np.ndarray):
```

Comment:
can you add some comments here on what is changed, it is non-obvious from inspection what is changing (e.g. what is true in prior version and what is true now)

Reply:
yah as mentioned above, i have no frikkin idea what the underlying logic is here